src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -54,24 +56,23 @@
   private final ProcessRunner processRunner;
   private final String appCommandMetricsEnvironment;
   private final String appCommandMetricsEnvironmentVersion;
-  private final Integer appCommandGsUtil;
-  private final File appCommandCredentialFile;
+  @Nullable private final File appCommandCredentialFile;
```

Comment:
Nullable looks odd here because we're not using it anywhere else, although some other fields are nullable too.